src/catalogue/application/produit/__tests__/produitController.spec.ts

```diff
@@ -2,37 +2,30 @@ import fs from 'fs'
 import { makeExecutableSchema } from 'graphql-tools'
```

Comment:
On a un warning qui conseille d'utiliser ça comme import plutôt
```suggestion
import { makeExecutableSchema } from '@graphql-tools/schema'
```